### PR TITLE
Update vagranttutorial.md (fixes #698)

### LINKED ISSUE #698: How to create another VM
### Problem
Since I already had a Virtual Machine already running because of the myPlanet team, it took me a little bit to figure out what command to use to create a new Virtual Machine. 

 ### Steps to reproduce the problem
Have already created a vm, and then try to use the steps to create a new one. 

 ### Screenshots
N/A

 ### Proposed solution
Add information on how to get around this issue at the bottom of Step 4 page (Vagrant Tutorial)

### PR DESCRIPTION
**Pull request by gautamsethi1**
Fixes #698 

### Description
Adding troubleshooting instructions if prospective interns already have a VM running.

### Raw.Githack preview link
https://raw.githack.com/gautamsethi1/gautamsethi.github.io/adding-instructions/#!pages/vi/vagranttutorial.md

### CODE REVIEW

**Review by anthonyhvo12 — CHANGES_REQUESTED** [review]
This is a great addition. I think putting the note at the beginning is more helpful as other interns don't have to worry about the rest of the steps.

**Comment by gautamsethi1** [comment]
> This is a great addition. I think putting the note at the beginning is more helpful as other interns don't have to worry about the rest of the steps.

Would this be a better spot? 

Screen Shot 2020-08-21 at 4 23 31 PM

**Comment by anthonyhvo12** [comment]
image
I think we can add the note right after "move on to the next steps" here.

File: pages/vi/vagranttutorial.md [hunk 1]
```diff
@@ -150,6 +150,11 @@ or not commonly used. To see all subcommands, run the command
 
 **NOTE**: At the end of this section, when you successfully installed `vagrant` and configured `cli`. Execute `vagrant global-status` command and post the code of your terminal to the [Gitter chat](https://gitter.im/treehouses/Lobby).
 
+## Troubleshooting
+
+What if I already have a Virtual Machine running with VirtualBox?
+- Use the command `vagrant up [name of cli folder]`
```
Comment (dogi) [inline pages/vi/vagranttutorial.md:156]: this is kinda wrong

**Pull request closed without merging**